crates/fuel-core/src/state.rs

```diff
@@ -33,11 +33,7 @@ where
 
 pub trait TransactableStorage<Height>: IterableStore + Debug + Send + Sync {
     /// Commits the changes into the storage.
-    fn commit_changes(
-        &self,
-        height: Option<Height>,
-        changes: Changes,
-    ) -> StorageResult<()>;
+    fn commit_changes(&self, changes: Changes) -> StorageResult<()>;
```

Comment:
We need `height: Option<Height>` for the state rewind feature=) I'm working on it right now, so I need this argument=D

I think maybe it makes sense to treat block height `None` as a regenesis and, at the end of the regenesis, set the block heights with information from the genesis block.

Reply:
It would have been nice to leave a comment indicating that the useless argument was there for a reason.

Reply:
Restored in 451f8be5d